packages/fuels/src/configs.ts

```diff
@@ -0,0 +1,5 @@
+export * from '@fuel-ts/transactions/configs';
+export * from '@fuel-ts/math/configs';
+export * from '@fuel-ts/program/configs';
+export * from '@fuel-ts/wallet/configs';
+export * from '@fuel-ts/address/configs';
```

Comment:
Maybe I couldn't express this clearly, but `fuels` is the only package that doesn't need a separate entry point for configs. At this point, there is no more risk of cyclic dependencies. All exports can be safely done on `fuels/src/index.ts`.

Reply:
Or you can have a `fuels/src/configs.ts` like the others.

And then `export * from './configs'` in the index.

Reply:
Gotcha @arboleya, thanks for pointing this out